### PR TITLE
ign -> gz : Remove redundant namespace references

### PR DESCRIPTION
**Pull request by methylDragon**
Part of: https://github.com/gazebo-tooling/release-tools/issues/784

### CODE REVIEW

File: examples/robot.cc [hunk 1]
```diff
@@ -327,7 +327,7 @@ int main(int argc, char *argv[])
         paths.FindSharedLibrary(robotLib));
 
   std::unordered_set<std::string> drivePlugins =
-      loader.PluginsImplementing("ignition::plugin::examples::Drive");
+      loader.PluginsImplementing("Drive");
```
Comment (chapulina) [inline examples/robot.cc:330]: These ones need to stay, they're strings

File: test/integration/factory.cc [hunk 2]
```diff
@@ -45,36 +45,36 @@ TEST(Factory, Construct)
   ignition::plugin::Loader pl;
   pl.LoadLib(IGNFactoryPlugins_LIB);
 
-  auto nameFactory = pl.Factory<NameFactory>("test::util::DummyNameForward");
```
Comment (chapulina) [inline test/integration/factory.cc:48]: These are all strings too

**Review by chapulina — CHANGES_REQUESTED** [review]

File: test/integration/factory.cc (hunk 2 above)
Comment (methylDragon) [reply test/integration/factory.cc]: Fixed! https://github.com/gazebosim/gz-plugin/compare/1c880de399144e1efd8f2a4afebc8dbea150b7c7..97fd2e82b0f72bb87f1129e9de8a76c5aaf7e2a3

File: examples/robot.cc (hunk 1 above)
Comment (methylDragon) [reply examples/robot.cc]: Fixed! https://github.com/gazebosim/gz-plugin/compare/1c880de399144e1efd8f2a4afebc8dbea150b7c7..97fd2e82b0f72bb87f1129e9de8a76c5aaf7e2a3

**Comment by methylDragon** [comment]
It is now a `ONE LINE PULL REQUEST` :rofl:

**Review by chapulina — APPROVED** [review]
😄

**Pull request merged**